cinema_service/settings.py

```diff
@@ -20,14 +24,13 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.environ.get("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.environ.get("DEBUG")
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS", "").split(",") \
+    if os.environ.get("ALLOWED_HOSTS") else []
```

Comment:
The construction of the ALLOWED_HOSTS list is a bit complex and can be simplified. Consider using the .get() method with a default value of an empty string (''), and then split the result. This will return an empty list if the environment variable is not set.